curator-framework/src/main/java/org/apache/curator/framework/imps/NamespaceWatcherMap.java

```diff
@@ -70,6 +70,14 @@ NamespaceWatcher remove(Object key)
     {
         return map.remove(key);
     }
+    
+    /**
+     * Remove all watchers for a given path
+     * @param path
+     */
+    void removeAllForPath(String path) {
+        
```

Comment:
Looks like I started doing something with the NamespaceWatcherMap and then realised that I shouldn't. Will remove this method, it's not used anywhere.